### ISSUE TITLE
Get something done for the UI

### ISSUE THREAD

**Issue by CharlieADavies** [CharlieADavies/code-jam-6#4]
We have a wireframe for the UI. Its probably a good idea to figure out what widgets we're going to need, and start working on them. Also deciding on things like how we're going to do the layout.

I imagine the first step is just rendering a very basic version of the card widget (A rectangley thing of some sort) onto a grid layout.

**Comment by glowingrunes** [comment]
Originally I thought of using a basic Rectangle, adding rounded edges and filling it with an image (which would be a stone texture). But given the nice image you found for the stone tablet, we could work off of the exercise we did for the Kivy code jam qualifier and use an Image instead of a Rectangle. You could interact with it either by dragging the mouse over it left or right, or to make it simpler maybe just use the left and right arrow keys, for the main stone tablet in the middle. The deck of cards would just need to be clicked on, and all the other stone tablets would need no interaction from the user.

TL;DR: Rectangles or Images 😛 Maybe a Canvas for the background?
